lib/AFS/CellCC/Dump.pm

```diff
@@ -254,40 +264,63 @@ _do_dump($$$) {
                               cb => sub {
             my ($interval) = @_;
 
-            my ($pretty_bytes, $pretty_rate) = describe_file($dump_fh->filename,
+            my ($pretty_bytes, $pretty_rate) = describe_file($dump_fhs[0]->filename,
                                                              \$last_bytes,
                                                              \$last_time);
 
-            my $descr = "Running vos dump ($pretty_bytes / $pretty_total dumped, $pretty_rate)";
-            update_job(jobid => $job->{jobid},
-                       dvref => \$job->{dv},
-                       from_state => $state,
-                       timeout => $interval+60,
-                       description => $descr);
+            $descr = "Running vos dump ($pretty_bytes / $pretty_total dumped, $pretty_rate)";
+            for my $i (0 .. $#jobs) {
```

Comment:
these `update_job` calls should also be wrapped by `db_rw`

updating status for all "child" jobs also seems unnecessary, but I guess we need to make sure they don't timeout... so oh well